<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>toStorageTablespaceUI</class>
  <widget class="QWidget" name="toStorageTablespaceUI" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>592</width>
        <height>480</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Form1</string>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item row="0" column="0" >
        <widget class="toFilesize" name="MinimumExtent" />
      </item>
      <item row="1" column="0" >
        <widget class="Q3ButtonGroup" name="ButtonGroup1" >
          <property name="title" >
            <string>&amp;Options</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="0" column="0" >
              <widget class="QCheckBox" name="Logging" >
                <property name="text" >
                  <string>&amp;Logging</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="toolTip" >
                  <string>Should objects created in this tablespace be logging by default</string>
                </property>
              </widget>
            </item>
            <item row="2" column="0" >
              <widget class="QCheckBox" name="Online" >
                <property name="text" >
                  <string>&amp;Online</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="toolTip" >
                  <string>Should tablespace be online at the time of creation</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QCheckBox" name="Permanent" >
                <property name="text" >
                  <string>&amp;Permanent</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="toolTip" >
                  <string>Should objects in this tablespace be permanent after the end of the session</string>
                </property>
              </widget>
            </item>
            <item row="3" column="0" >
              <widget class="QCheckBox" name="DefaultStorage" >
                <property name="text" >
                  <string>&amp;Default storage</string>
                </property>
                <property name="toolTip" >
                  <string>Specify default storage clause (In separate tab)</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="Q3ButtonGroup" name="ExtentGroup" >
          <property name="title" >
            <string>&amp;Extent allocation</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="0" column="0" >
              <widget class="QRadioButton" name="Dictionary" >
                <property name="text" >
                  <string>&amp;Dictionary</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="toolTip" >
                  <string>Manage tablespace extents using dictionary tables</string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QRadioButton" name="LocalAuto" >
                <property name="text" >
                  <string>Local &amp;autoallocation</string>
                </property>
                <property name="toolTip" >
                  <string>Specify how space is allocated in the tablespace</string>
                </property>
              </widget>
            </item>
            <item row="2" column="0" >
              <widget class="QRadioButton" name="LocalSelect" >
                <property name="enabled" >
                  <bool>true</bool>
                </property>
                <property name="text" >
                  <string>Local &amp;uniform</string>
                </property>
                <property name="toolTip" >
                  <string>The tablespace is locally managed with fixed extent sizes of specified size</string>
                </property>
              </widget>
            </item>
            <item row="3" column="0" >
              <widget class="toFilesize" name="LocalUniform" >
                <property name="enabled" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="3" column="0" >
        <spacer name="Spacer1" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>toFilesize</class>
      <extends>QWidget</extends>
      <header location="local" >tofilesize.h</header>
      <sizehint>
        <width>200</width>
        <height>40</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>1</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0" >
      <data format="XPM.GZ" length="646" >789c6dd2c10ac2300c00d07bbf2234b7229d1be245fc04c5a3201e4615f430059d0711ff5ddb2e6bb236ec90eed134cb5a19d8ef36602af5ecdbfeeac05dda0798d3abebde87e3faa374d3807fa0d633a52d38d8de6f679fe33fc776e196f53cd010188256a3600a292882096246517815ca99884606e18044a3a40d91824820924265a7923a2e8bcd05f33db1173e002913175f2a6be6d3294871a2d95fa00e8a94ee017b69d339d90df1e77c57ea072ede6758</data>
    </image>
  </images>
  <connections>
    <connection>
      <sender>DefaultStorage</sender>
      <signal>toggled(bool)</signal>
      <receiver>toStorageTablespaceUI</receiver>
      <slot>allowDefault(bool)</slot>
    </connection>
    <connection>
      <sender>Permanent</sender>
      <signal>toggled(bool)</signal>
      <receiver>toStorageTablespaceUI</receiver>
      <slot>permanentToggle(bool)</slot>
    </connection>
    <connection>
      <sender>Dictionary</sender>
      <signal>toggled(bool)</signal>
      <receiver>toStorageTablespaceUI</receiver>
      <slot>dictionaryToggle(bool)</slot>
    </connection>
    <connection>
      <sender>LocalSelect</sender>
      <signal>toggled(bool)</signal>
      <receiver>toStorageTablespaceUI</receiver>
      <slot>uniformToggle(bool)</slot>
    </connection>
  </connections>
</ui>
